packages/components/src/notice/stories/index.story.tsx

```diff
@@ -109,7 +109,11 @@ export const NoticeListSubcomponent: StoryFn< typeof NoticeList > = () => {
 	return (
 		<>
 			<NoticeList notices={ notices } onRemove={ removeNotice } />
-			<Button variant="primary" onClick={ resetNotices }>
+			<Button
+				__next40pxDefaultSize
```

Comment:
<img width="317" alt="Reset button in story" src="https://github.com/user-attachments/assets/d5b8da53-bc21-4e00-bfe4-d02fcad2db0c">

Reply:
Pre-existing, but isn't it weird that those notices have no margins by default?

Reply:
Not sure 🤷 This component is on the "needs design updates" docket though.